src/Shader.h

```diff
@@ -2,35 +2,34 @@
 #define SHADER_H
 
 #include <string>
+
 #include <GL/glew.h>
-#include "transform.h"
+
+#include "Transform.h"
 
 class Shader
 {
 public:
 	Shader(const std::string& fileName);
+	virtual ~Shader();
 	void Bind();
 	void Update(const Transform& transform);
-	virtual ~Shader();
 
-protected:
 private:
 	static const unsigned int NUM_SHADERS = 2;
-	Shader(const Shader& other) {}
-	void operator=(const Shader other) {}
```

Comment:
These should not have been removed. I do recommend changing the syntax to c++11 though.